index.ts

```diff
@@ -22,12 +22,23 @@ const app = express();
 // Desactivate debug log for production as they are a bit too verbose
 if (!config.activateDebug) {
   console.log('console.debug is not active - thanks to ACTIVATE_DEBUG_LOG env variable');
-  console.debug = (): void => {};
+  console.debug = (): void => { };
 }
 
 // HSTS is managed directly by scalingo => if we set it there it appears twice and can be misinterpreted
 app.use(helmet({ contentSecurityPolicy: false, hsts: false }));
 
+app.use((req, res, next) => {
+  res.setHeader(
+    'Permissions-Policy',
+    // Generated using https://www.permissionspolicy.com/
+    // All features were disabled
+    // eslint-disable-next-line max-len
+    'accelerometer=(), ambient-light-sensor=(), autoplay=(), battery=(), camera=(), cross-origin-isolated=(), display-capture=(), document-domain=(), encrypted-media=(), execution-while-not-rendered=(), execution-while-out-of-viewport=(), fullscreen=(), geolocation=(), gyroscope=(), magnetometer=(), microphone=(), midi=(), navigation-override=(), payment=(), picture-in-picture=(), publickey-credentials-get=(), screen-wake-lock=(), sync-xhr=(), usb=(), web-share=(), xr-spatial-tracking=()',
+  );
```

Comment:
@desoindx even with geolocalisation disabled (`geolocalisation=()`) I still can use it.. Maybe because it's on client side ?

Reply:
it's working for me
and the point of those permission are to managed what is done on the front side ! so it should :)